llarp/dns.cpp

```diff
@@ -96,7 +96,7 @@ getDNSstring(const char *const buffer, uint32_t *pos)
 }
 
 void
```

Comment:
this file is deprecated

Reply:
yeah, will rebase on top of #686 once that is merged